### PR TITLE
KOA-5066 Adding BPKCard for SwiftUI

### PR DESCRIPTION
**Pull request by frugoman**
Adding BPKCard component

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/main/Example/Backpack%20Screenshot/Screenshots.swift)
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/main/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Backpack-SwiftUI/BPKCard/Classes/BPKCard.swift [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ 
+import SwiftUI
+
+public struct BPKCard<Content: View>: View {
+    public enum CornerStyle {
+        case large, small
+    }
+    
+    @Binding public var isSelected: Bool
+    private let content: Content
```
Comment (gert-janvercauteren) [inline Backpack-SwiftUI/BPKCard/Classes/BPKCard.swift:27]: I would annotate like this
`@ViewBuilder let content: () -> Content` to allow more than 1 view to be added

File: Backpack/StarRating/Classes/BPKStar.m [hunk 2]
```diff
@@ -144,7 +144,7 @@ - (UIColor *)currentStarColor {
 - (BPKIconSize)iconSizeForStarSize:(BPKStarSize)size {
     switch (size) {
     case BPKStarSizeXLarge:
-        return BPKIconSizeXLarge;
+        return BPKIconSizeLarge;
```
Comment (marianeum) [inline Backpack/StarRating/Classes/BPKStar.m:147]: was that intentional?

File: Backpack-SwiftUI/Card/Classes/BPKCard.swift [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+import SwiftUI
+
+public struct BPKCard<Content: View>: View {
+    public enum CornerStyle {
+        case large, small
+    }
+    
+    private var isSelected: Bool
+    private let content: Content
+    private var padded: Bool
```
Comment (marianeum) [inline Backpack-SwiftUI/Card/Classes/BPKCard.swift:28]: one thing i might do is not use a boolean for all of these props, but an enum, so it's easier to extend - we did the same with padding in compose and wish we did it with focused given the talk about adding a flat style as well

File: Backpack-SwiftUI/Card/Classes/BPKCard.swift [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    private var cornerStyle: CornerStyle
+    private var backgroundColor: BPKColor = .backgroundTertiaryColor
+    private var tapAction : () -> Void = {}
+    
+    public init(
```
Comment (marianeum) [inline Backpack-SwiftUI/Card/Classes/BPKCard.swift:33]: do we have to add anything for accessibility here? how will this be announced/was anything custom done on the uikit version?

**Review by marianeum — APPROVED** [review]

**Pull request merged**